sample-code/javascript-wd/test/helpers/apps.js

```diff
@@ -1,4 +1,4 @@
-if (process.env.DEV) {
+if (process.env.DEV || process.env.SAUCE_LABS) {
```

Comment:
The README above says 
```
`DEV`: If set to true, runs apps that are in local directory, otherwise runs  apps that are downloaded from GitHub
```
but this is the opposite.

Reply:
I changed the README.md